Elizabeth_Theme/src/css/_custom.scss

```diff
@@ -0,0 +1,36 @@
+@import 'colors';
+@import 'components/buttons';
+@import 'components/map';
+@import 'layout/footer';
+@import 'layout/navigation';
+@import 'layout/navigation_scrolled';
+@import 'layout/welcome_page';
+
+body {
+    &.has-control-menu {
+        header {
+			margin-top: 0; 
+
+			@include media-breakpoint-down(sm) {
+			margin-top: 48px;
+			}  
+        }
+  	}
+
+  	.initial-background {
+		background-repeat: no-repeat;
+		background-size: cover;
+		background:
+			#3c3a3a
```

Comment:
If we have a `colors` SCSS file, why not put all the theme's colors in that file?